packages/carbon-web-components/src/components/ui-shell/header-nav-item.ts

```diff
@@ -35,18 +36,36 @@ class BXHeaderNavItem extends FocusMixin(LitElement) {
   @property()
   title!: string;
 
+  /**
+   * Applies selected styles to the item if a user sets this to true and `aria-current !== 'page'`.
+   */
+  @property({ type: Boolean, attribute: 'is-active', reflect: true })
+  isActive = false;
+
+  /**
+   * indicates that this element represents the current item
+   */
+  @property({ type: Boolean, attribute: 'aria-current', reflect: true })
+  ariaCurrent;
+
   /**
    * As child of <ul>, this element must have role of listitem
    */
   @property({ reflect: true })
   role: string = 'listitem';
 
   render() {
-    const { href, title } = this;
+    const { ariaCurrent, href, isActive, title } = this;
+    const linkClass = classMap({
+      [`${prefix}--header__menu-item`]: true,
+      [`${prefix}--header__menu-item--current`]:
+        isActive && ariaCurrent !== 'page',
```

Comment:
Just a little confused, should `ariaCurrent` be declared as a `type: String` (or no specific type) instead of a Boolean? Only since it's being compared to a string here.

Reply:
good catch! Sloppy copy and paste maybe 😬